muesli/web/templates/user/register.pt

```diff
@@ -5,14 +5,14 @@
 <h2>Bei MÜSLI registrieren</h2>
 
 <p>
-Um sich anzumelden füllen Sie bitte das Formular unten aus.  Bitte beachten Sie, daß die Daten später <strong>nicht mehr verändert werden können</strong>.
+Um sich anzumelden, füllen Sie bitte das Formular unten aus. Bitte beachten Sie, dass die Daten später <strong>nicht mehr verändert werden können</strong>.
 </p>
 
 <p>
 Assistenten und Übungsleiter, die <strong>nicht</strong> selbst an Übungsgruppen teilnehmen, können sich auch <a tal:attributes="href request.route_path('user_register_other')">mit weniger Angaben anmelden</a>.
-Werden Rechte zum Eintragen von Vorlesungen benötigt so <a tal:attributes="href request.route_path('contact')">melden Sie sich bitte bei uns</a> nach der Anmeldung.
+Werden Rechte zum Eintragen von Vorlesungen benötigt, so <a tal:attributes="href request.route_path('contact')">melden Sie sich bitte bei uns</a> nach der Anmeldung.
 <br/><br/>
-Verwenden Sie eine Mailadresse unter <strong>uni-heidelberg.de</strong>, beispielsweise &lt;uniid&gt;@stud.uni-heidelberg.de . Falls Sie noch keine besitzen, benutzen Sie bitte eine andere und ändern Sie diese sobald eine uni-heidelberg.de Adresse vorliegt.
+Verwenden Sie eine Mailadresse unter <code>uni-heidelberg.de</code>, beispielsweise <strong><code>&lt;uniid&gt;@stud.uni-heidelberg.de</code></strong>. Falls Sie noch keine besitzen, benutzen Sie bitte eine andere und ändern Sie diese sobald möglich.
```

Comment:
Having one in `<strong>` and the other without looks a bit weird to me on the UI.

Reply:
This is again intentional: I want to direct the attention of a user to `<uniid>@stud.uni-heidelberg.de` so that their first association is "Oh, I recognize that, I'll type in the usual uni email".

The explanation should be less dominant because it is only interesting for those that want to know the exact requirements (and caused problems in the past).